src/components/ColorPicker.scss

```diff
@@ -99,7 +99,7 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  z-index: 1;
+  z-index: 0;
```

Comment:
if we go with `0` we should just remove it.. as it's the default one.

Reply:
@lipis this makes sense. Can't believe I missed it!